packages/components/atoms/f-form-field/src/components/FormSelectionControl.vue

```diff
@@ -0,0 +1,187 @@
+                }]"
+            @change="updateSelectionControl"
+        >
+
+        <form-label
+            :id="`label-${$attrs.id}`"
+            :label-for="$attrs.id"
+        >
+            {{ labelText }}
+        </form-label>
+    </div>
+</template>
+
+<script>
+import FormLabel from './FormLabel.vue';
+import {
+    VALID_SELECTION_CONTROL_TYPES,
+    DEFAULT_SELECTION_CONTROL_TYPE
+} from '../constants';
+
+export default {
+    components: {
+        FormLabel
+    },
+
+    inheritAttrs: false,
+
+    props: {
+        labelText: {
+            type: String,
+            default: ''
+        },
+        attributes: {
+            type: Object,
+            default: () => {}
+        },
+        inputType: {
+            type: String,
+            default: DEFAULT_SELECTION_CONTROL_TYPE,
+            validator: value => VALID_SELECTION_CONTROL_TYPES.indexOf(value) !== -1
+        },
+        value: {
+            type: String,
+            default: ''
+        },
+        hasError: {
+            type: Boolean,
+            default: false
+        }
+    },
+
+    computed: {
+        testId () {
+            const formFieldName = (this.attributes && this.attributes.name ? this.attributes.name : null);
+
+            return {
+                container: formFieldName ? `formfield-${formFieldName}` : 'formfield-container',
+                input: formFieldName ? `formfield-${formFieldName}-${this.inputType}` : 'formfield-input'
+            };
+        },
+
+        isCheckbox () {
+            return this.inputType === 'checkbox';
+        }
+    },
+
+    methods: {
+        updateSelectionControl (event) {
+            this.$emit('update', event.target.checked);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+    /**
+        Changing the color of an inline SVG. The color needs to be encoded, so '#' turns into '%23'.
+        Had a lot of problems trying to save the SVG in the `assets` folder, since it wouldn't load, and including it
+        in `f-vue-icons` wouldn't help, since we need to use this in the `<style>` section, and not as a Vue component.
+    */
+    @mixin tick-svg($color) {
+        $encodedColor: '%23' + str-slice(#{$color}, 2);
+        background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg fill='none' viewBox='0 0 15 12' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m4.5 9.4722-2.9861-2.9861c-0.15636-0.1569-0.36876-0.2451-0.59027-0.2451-0.22152 0-0.43392 0.0882-0.59028 0.2451-0.15691 0.15637-0.2451 0.36877-0.2451 0.59028 0 0.22152 0.088195 0.43392 0.2451 0.59028l3.5771 3.5771c0.15636 0.1569 0.36876 0.2451 0.59028 0.2451 0.22151 0 0.43391-0.0882 0.59028-0.2451l8.9229-8.9243c0.1569-0.15637 0.2451-0.36877 0.2451-0.59028 0-0.22152-0.0882-0.43392-0.2451-0.59028-0.1564-0.15691-0.3688-0.2451-0.5903-0.2451s-0.4339 0.088195-0.5903 0.2451l-8.3333 8.3333z' clip-rule='evenodd' fill='#{$encodedColor}' fill-rule='evenodd'/%3E%3C/svg%3E");
```

Comment:
One possible improvement (definitely a separate ticket, rather than as part of this one), could be to use something like [postcss-inline-svg](https://www.npmjs.com/package/postcss-inline-svg) to inline the SVG here – you could then include the image from f-vue-icons (as they should be available from the `dist` folder that gets published).

Reply:
Will create a ticket to investigate this 👍